package.json

```diff
@@ -87,6 +87,7 @@
     "firefox-client": "0.3.0",
     "flow-bin": "0.33.0",
     "git-hooks": "1.1.1",
```

Comment:
We can remove this. I just tried it on Mac and it works just fine with only the `git-hooks-win` package.